src/app/layout/Layout.scss

```diff
@@ -75,7 +84,7 @@
 			border-top: solid 1px $gray-800;
 		}
 		&_topSection {
-			display: flex;
+			display: fix;
```

Comment:
why is it needed to change that to fix?

Reply:
when experimenting with fixing the navbar in the phone view I must have missed to restore that